### PR TITLE
Implement support for advanced queries

### PR DESCRIPTION
**Pull request by stevekinney**
Adds very basic support for advanced queries.

### CODE REVIEW

File: src/global.d.ts [hunk 1]
```diff
@@ -59,13 +59,11 @@ type FilterParameters = {
...
+  query?: string;
 };
 
-type ArchiveFilterParameters = {
-  workflowId?: string;
-  workflowType?: string;
-  closeTime?: Duration | string;
-  executionStatus?: WorkflowStatus;
+type ArchiveFilterParameters = Omit<FilterParameters, 'timeRange'> & {
+  closeTime: Duration | string;
```
Comment (GiantRobots) [inline src/global.d.ts:66]: 👍🏾

File: src/lib/utilities/request-from-api.ts [hunk 2]
```diff
@@ -3,13 +3,13 @@ import { handleError } from './handle-error';
 import { isFunction } from './is-function';
 import { toURL } from './to-url';
 
-type TemporalAPIError = {
+export type TemporalAPIError = {
   code: number;
   message: string;
   details: unknown[];
 };
```
Comment (GiantRobots) [inline src/lib/utilities/request-from-api.ts:10]: These may be useful globally

Comment (stevekinney) [reply src/lib/utilities/request-from-api.ts:10]: I totally agree.

File: src/routes/namespaces/[namespace]/workflows/index.svelte [hunk 3]
```diff
@@ -52,17 +62,14 @@
...
       <VirtualList items={workflows} let:item>
         <WorkflowsSummaryRow workflow={item} {timeFormat} />
       </VirtualList>
     </WorkflowsSummaryTable>
   {:else}
-    <EmptyState
-      title={'No Workflows Found'}
-      content={'If you have filters applied, try adjusting them.'}
-    />
+    <EmptyState title={'No Workflows Found'} content={errorMessage} />
```
Comment (GiantRobots) [inline src/routes/namespaces/[namespace]/workflows/index.svelte:73]: 👌

File: src/routes/namespaces/[namespace]/workflows/_workflow-filters.svelte [hunk 4]
```diff
@@ -21,32 +20,73 @@
...
-      <Option {value}>{label}</Option>
-    {/each}
-  </FilterSelect>
-  <Select id="filter-by-relative-time" bind:value={timeFormat}>
-    <Option value={'relative'}>Relative</Option>
-    <Option value={'UTC'}>UTC</Option>
-    <Option value={'local'}>Local</Option>
-  </Select>
-</div>
+<section class="flex flex-col gap-2">
```
Comment (GiantRobots) [inline src/routes/namespaces/[namespace]/workflows/_workflow-filters.svelte:40]: I can see an eventuality when this becomes it's own component but this is good for now.

**Review by GiantRobots — APPROVED** [review]

File: src/routes/namespaces/[namespace]/workflows/_workflow-filters.svelte (hunk 4 above)
Comment (GiantRobots) [reply src/routes/namespaces/[namespace]/workflows/_workflow-filters.svelte:40]: Man my brain is dumb this is already it's own component.

**Pull request merged**